src/restaurant_inspections_api/handlers/businesses.clj

```diff
@@ -13,6 +13,13 @@
                              :perPage (validate/per-page per-page)
                              :page (validate/page page)}))
 
+(defn format-params
+  [params-map]
+  (assoc params-map
+         :zipCodes (when-let [zipCodes (:zipCodes params-map)]
```

Comment:
this when-let var (zipCodes) should be replaces with zip-codes, since it is not the key for the map. in fact, is when-let necessary here?